packages/devtools_app/lib/src/screens/memory/shared/primitives/instance_context_menu.dart

```diff
@@ -15,14 +14,11 @@ class InstanceViewWithContextMenu extends StatelessWidget {
   const InstanceViewWithContextMenu({
     super.key,
     required this.count,
-    required this.gaContext,
     required this.menuBuilder,
   }) : assert(count >= 0);
 
   final int count;
   final MenuBuilder? menuBuilder;
-  final MemoryAreas gaContext;
-
```

Comment:
@polina-c this seems like a bug that this is unused?

Reply:
Should I add a todo or can you suggest what changes are expected to fix it?

Reply:
@polina-c please advise

Reply:
I see that this has already been addressed, I'm not adding it then